imagersite/imager_profile/views.py

```diff
@@ -1,7 +1,30 @@
 from django.shortcuts import render
 from django.contrib.auth.decorators import login_required
+from django.http import HttpResponseRedirect
+from .forms import *
 
 
 @login_required
 def profile_view(request):
-    return render(request, 'profile.html')
\ No newline at end of file
+    return render(request, 'profile.html')
+
+
+@login_required
+def edit_profile_view(request):
+    if request.method == 'POST':
+        profile_form = ProfileForm(request.POST, instance=request.user.profile)
+        user_form = UserForm(request.POST, instance=request.user)
```

Comment:
processing two forms in parallel is a perfectly good way to approach this problem.  It's not easy to do with CBVs but not too hard this way.  Good.